src/Acorn/Bootstrap/SageFeatures.php

```diff
@@ -6,35 +6,39 @@
 
 class SageFeatures
 {
+    /**
+     * Sage Features
+     *
+     * @var array
+     */
+    protected $features = [
+        'sage' => \Roots\Acorn\Sage\SageServiceProvider::class,
```

Comment:
`sage` should always get loaded if either `assets` or `blade` are loaded.

Reply:
fixed & cleaned up